docs/DeveloperGuide.adoc

```diff
@@ -153,7 +152,68 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-// tag::undoredo[]
+=== Class particulars feature
+The class particulars feature keeps track of essential student details.
+The feature comprises of four commands namely,
+
+* `AddCommand` - Adds the student particulars into the class list
+* `EditCommand` - Edits the particulars of a student
+* `DeleteCommand` - Deletes the student information
+* `FindCommand` - Finds information of the required student
+
+The commands when executed, will interface with the methods exposed by the `Model` interface to perform the related operations
+(See <<Design-Logic,logic component>> for the general overview).
+
+.Class Diagram for Commands
```

Comment:
Class diagram doesn't seem to match the theme of the other UML diagrams

Reply:
Perhaps you could add some navigability and methods in the class diagram to better explain the respective commands